gpflow/kernels/base.py

```diff
@@ -117,6 +120,24 @@ def slice_cov(self, cov: tf.Tensor) -> tf.Tensor:
 
         return cov
 
+    def validate_ard_parameter(self, X, parameter):
```

Comment:
Could also check whether `active_dims` has the right length (if not None)?

Reply:
I will add in a check on __init__ to ensure size of `active_dims` has the right length (if not None and if not a slice)